website/content/en/preview/contributing/development-guide.md

```diff
@@ -101,47 +128,28 @@ While you can tail Karpenter's logs with kubectl, there's a number of tools out
 stern -n karpenter -l app.kubernetes.io/name=karpenter
 ```
 
-## Environment specific setup
-
-### AWS
-
-For local development on Karpenter you will need a Docker repo which can manage your images for Karpenter components.
-You can use the following command to provision an ECR repository. We recommend using a single "dev" repository for 
-development across multiple projects, and to use specific image hashes instead of image tags. 
-
-```bash
-aws ecr create-repository \
-    --repository-name dev \
-    --image-scanning-configuration scanOnPush=true \
-    --region "${AWS_DEFAULT_REGION}"
-```
-
-Once you have your ECR repository provisioned, configure your Docker daemon to authenticate with your newly created repository.
-
-```bash
-export KO_DOCKER_REPO="${AWS_ACCOUNT_ID}.dkr.ecr.${AWS_DEFAULT_REGION}.amazonaws.com/dev"
-aws ecr get-login-password --region "${AWS_DEFAULT_REGION}" | docker login --username AWS --password-stdin "${KO_DOCKER_REPO}"
-```
-
 Finally, to deploy the correct IAM permissions, including the instance profile for provisioned nodes, run
 
 ```bash
 make setup
 ```
 
 ## Profiling memory
+
 Karpenter exposes a pprof endpoint on its metrics port.
 
 Learn about profiling with pprof: https://jvns.ca/blog/2017/09/24/profiling-go-with-pprof/
 
 ### Prerequisites
```

Comment:
Maybe prerequisites can move up higher in the doc and be more inclusive of all the components needed in this page.